dashboards-observability/.cypress/integration/1_event_analytics.spec.js

```diff
@@ -911,3 +911,74 @@ describe('Renders heatmap chart for Chart Style', () => {
     cy.get('stop[stop-color="rgb(214, 191, 87)"]').should('exist');
    });
 });
+
+describe('Renders Tree Map for Parent Fields ', () => {
+  beforeEach(() => {
+    landOnEventVisualizations();
+  });
+
+  it('Renders Tree Map and Add Single Parent', () => {
+    querySearch(TEST_QUERIES[7].query, TEST_QUERIES[7].dateRangeDOM);
```

Comment:
For single parent test case can common for below test cases.

Reply:
removed this test case